app/views/businesses/index.html.erb

```diff
@@ -9,7 +9,9 @@
 <%= link_to 'Neues Business anlegen',
             new_business_path,
             :class => 'btn btn-primary' %>
-<br>
+<%= link_to 'Business Kategorien von google übernehmen',
```

Comment:
Es macht wahrscheinlich mehr Sinn, den Button nicht so präsent darzustellen, da der Link immer alle Businesses bereinigt.
Vorschlag:
Wir nennen den Link in der Navbar `Dead Link Checker` um zu `Einmalige Datenbereinigung` und machen da einen Abschnitt `Business Kategorien von Google übernehmen` mit dem Button